Resources/Prototypes/_White/Entities/Structures/Storage/Canisters/gas_canisters.yml

```
@@ -0,0 +1,739 @@
+          AntiNobliumCanisterBroken:
+            min: 1
+            max: 1
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+      - !type:DumpCanisterBehavior
+  - type: Lock
+    locked: true
+
+# Broke Entities
+
+- type: entity
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 0

(indentation)